MSW/MSW/Program.cs

```diff
@@ -8,36 +8,49 @@ class Program
 	{
 		//TODO: add firewall rules on the required ports. 
 		/*INetFwRule firewallRule = (INetFwRule)Activator.CreateInstance(
-    Type.GetTypeFromProgID("HNetCfg.FWRule"));
+	Type.GetTypeFromProgID("HNetCfg.FWRule"));
 firewallRule.Action = NET_FW_ACTION_.NET_FW_ACTION_ALLOW;
 firewallRule.Description = "Enables eATM REST Web Service adapter       
-    traffic.";
```

Comment:
This is a change inside a string. Is this intended? Not sure how much this affects anything, I'm not familiar with this interface.

Reply:
Some editor settings did this , not intensional. Might not be an issue, but I will revert it to the old value just in-case